js/scripts/dimension/playground_dimension.js

```diff
@@ -92,9 +92,10 @@ const loadPlaygroundDimension = () => {
         GAME.addEntity(new DialogBubble(CHAD, "Hey pal! My name's Papa Chad", DialogBubble.NORMAL));
         GAME.addEntity(new Crosshair());
         GAME.addEntity(new Slingshot());
- 
-        CHAD.x = -3 * Block.SCALED_SIZE;
-        CHAD.y = -20 * Block.SCALED_SIZE;
+
+        // CHAD.x = -3 * Block.SCALED_SIZE;
+        // CHAD.y = -20 * Block.SCALED_SIZE;
+        CHAD.pos = Vector.blockToWorldSpace(new Vector(-3, -20));
```

Comment:
What does this do? I am lost a bit. @NathanHinthorne

Reply:
`Vector.blockToWorldSpace` just translates the passed coordinate vector (which, since we are treating it as block coordinates, represents -3 blocks on the x-axis and -20 blocks on the y-axis) from block space to world space. It is performing the exact same operation as in the commented-out code above except using Vectors instead of separate x/y fields.

Reply:
Thanks @tclaar!